core/errors/packager.h

```diff
@@ -10,7 +10,7 @@ constexpr ErrorClass InvalidPackageDefinition{3702, StrictLevel::False};
 constexpr ErrorClass RedefinitionOfPackage{3703, StrictLevel::False};
 constexpr ErrorClass PackageNotFound{3704, StrictLevel::False};
 constexpr ErrorClass UnpackagedFile{3705, StrictLevel::False};
-constexpr ErrorClass InvalidImportOrExport{3706, StrictLevel::False};
+constexpr ErrorClass InvalidConfiguration{3706, StrictLevel::False};
```

Comment:
Is it worth either having separate error messages, or updating the existing error docs to talk about this new failure mode?

Reply:
I don't think this new failure mode is worth documenting publicly. It's very niche within our packaging system (which is already only used at Stripe), and it's not a first-class feature. I wanted to reuse the same `ErrorClass`, hence why I renamed it internally -- if you feel strongly that this has to be a brand new error code then I'm happy to make it so!

Reply:
I don't really mind whether it's one or two error codes, but I think that

- the time when we introduce the new behavior is the best time to document it
- using "it's only in use at Stripe" as a reason for why not to document something just deprives _people at Stripe_ from having access to documentation

Reply:
Fair. I'll attempt to make a doc update here and see if it fits.

Reply:
done